pytorch_lightning/core/optimizer.py

```diff
@@ -58,7 +58,7 @@ def optimizer(self) -> Optimizer:
         return self._optimizer
 
     @classmethod
-    def _to_lightning_optimizer(
```

Comment:
Renamed since it's also used in Lite now

Reply:
But the variable names inside the method mentions lightning optimizer. Maybe need to update them too?

Reply:
What do you mean? The output is still a LightningOptimizer.

I just renamed it because it was weird to call

`_LiteOptimizer._to_lightning_optimizer`